src/tile_mate/tile_model.py

```diff
@@ -1,6 +1,11 @@
 import geopandas as gpd
 from pandera import Column, DataFrameSchema
 
+# Add support for GeometryDtype
+pandas_engine.Engine.dtype = staticmethod(
+    lambda dtype: dtype if dtype == gpd.array.GeometryDtype else None
+)
+
```

Comment:
Why are you writing this yourself? I don't understand this function - it returns True or `None`? 

Why not what they recommend: https://pandera.readthedocs.io/en/stable/geopandas.html

Reply:
Maybe it's just about using `pa.Column("geometry")` and pandera takes care of all that for you. That's what I would recommend.

I am concerned we are loosing out on the validation with this.

Reply:
@cmarshak good call. Yeah, I had no particular reason going about it with my original proposed changes, but I've now modified it by using pa.Column("geometry") directly to preserve validation.

Looks like it's also backwards compatible with the older `pandera` version.